Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM alpine:3.3
+
+MAINTAINER PRX <sysadmin@prx.org>
+
+RUN echo "http://nl.alpinelinux.org/alpine/edge/testing" >> /etc/apk/repositories && \
```

Comment:
Edge is the only place to get `autossh`.  This branch still supports ssh-tunneling mysql, even though that doesn't seem viable at this point.  Shall I gut that part, for now?  Or hold out hope that it'll get faster someday?